src/main/java/io/cdap/plugin/ariba/source/metadata/proto/Properties.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.ariba.source.metadata.proto;
+
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import io.cdap.plugin.ariba.source.util.ResourceConstants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Get Properties for all type of fields
+ */
+public class Properties {
+  JsonElement properties;
```

Comment:
where is this field set?

Reply:
Fixed here (https://github.com/data-integrations/ariba-plugins/pull/2/commits/9e5462f14445ea32445eb3bdd766d3d83f76499b)